app/components/dialogs/settings/SettingsDialog.tsx

```diff
@@ -227,6 +231,28 @@ const SettingsDialog = (props: Props) => {
         />
       )}
 
+      {isResetSettingsDialogOpened && (
+        <ConfirmDialog
+          open={isResetSettingsDialogOpened}
+          onClose={() => {
+            setIsResetSettingsDialogOpened(false);
+          }}
+          title="Confirm"
+          content={i18n.t('core:confirmResetSettings')}
+          confirmCallback={result => {
+            if (result) {
+              const electron = window.require('electron');
+              const webContents = electron.remote.getCurrentWebContents();
+              webContents.session.clearStorageData();
+              webContents.reload();
```

Comment:
this should work also on Cordova and Web versions

Reply:
Yes resetSettings button is displayed only in Electron